src/main/java/wolox/training/models/Book.java

```diff
@@ -46,9 +46,9 @@ public Book() {
 
     }
 
-    public Book(String genre, String autor, String image, String title, String subtitle, String publisher, String year, Integer pages, String isbn) {
+    public Book(String genre, String author, String image, String title, String subtitle, String publisher, String year, Integer pages, String isbn) {
 this.genre = genre;
-        this.autor = autor;
+        this.author = author;
```

Comment:
If you are going to do a modification that it's not in the card, the ideal is generate other branch with change, because 
this way it won't match with de card mentioned

Reply:
Perfect...thank you so much